kasten/kasten-generate-example-backup-policy/kasten-generate-example-backup-policy.yaml

```
@@ -0,0 +1,84 @@
+# This is an example rule intended to be cloned & modified to meet
+# organizational requirements. The `dataprotetion` label value
+# can be changed to correspond with specific policy templates.
+# 
+# The Kyverno background controller requires
+# additional permissions to create Kasten Policy resources.
+# Apply the create-kasten-policies-clusterrole.yaml manifest
+# first to grant the required permissions.
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: kasten-generate-example-backup-policy
+  annotations:
+    policies.kyverno.io/title: Generate Kasten Backup Policy Based on Resource Label
+    policies.kyverno.io/category: Veeam Kasten
+    kyverno.io/kyverno-version: 1.12.1
+    policies.kyverno.io/minversion: 1.12.0
+    kyverno.io/kubernetes-version: "1.24-1.30"
+    policies.kyverno.io/subject: Policy
+    policies.kyverno.io/description: >-
+      Generates a Kasten policy for a namespace that includes any Deployment or StatefulSet with a "dataprotection=kasten-example" label, if the policy does not already exist.
+
+      Use with "kasten-data-protection-by-label" policy to require "dataprotection" labeling on workloads.
```

When using a folded line (block) scalar like this, you can't have an empty line or else it breaks rendering.
```suggestion
      Generates a Kasten policy for a namespace that includes any Deployment or StatefulSet with a "dataprotection=kasten-example" label, if the policy does not already exist. Use with "kasten-data-protection-by-label" policy to require "dataprotection" labeling on workloads.
```